core/src/lib.rs

```diff
@@ -10,6 +10,7 @@ use std::sync::Arc;
 use bevy::ecs::schedule::ShouldRun;
 use bevy::prelude::*;
 
+use colliding_entities::CollidingEntities;
```

Comment:
`CollidingEntities::default()` is failing to resolve in my project. I tried adding it to heron::prelude, but it still didn't resolve until I change this line. Should this be public? 
```suggestion
pub use colliding_entities::CollidingEntities;
```

Reply:
Oh, right, thanks!